server/src/mysql/handler.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::{net::SocketAddr, sync::Arc};
+
+use catalog::manager::Manager as CatalogManager;
+use common_util::runtime::JoinHandle;
+use log::{error, info};
+use opensrv_mysql::AsyncMysqlIntermediary;
+use query_engine::executor::Executor as QueryExecutor;
+use snafu::ResultExt;
+use table_engine::engine::EngineRuntimes;
+use tokio::sync::oneshot::{self, Receiver, Sender};
+
+use crate::{
+    instance::{Instance, InstanceRef},
+    mysql::{error::*, worker::MysqlWorker},
+};
+pub struct MysqlService<C, Q> {
```

Comment:
- One newline above this line.
- Maybe the filename should be `service.rs` if `MysqlService` is used here.